src/Encamina.Enmarcha.SemanticKernel.Plugins.Text/Plugins/TextPlugin/KeyPhrasesLocaled/skprompt.txt

```diff
@@ -0,0 +1,11 @@
+The task is to EXTRACT JUST {{$topKeyphrases}} of the most significant keyphrases.
+ALWAYS return EXACTLY {{$topKeyphrases}} keyphares, no less and no more.
```

Comment:
I think that 'keyphares' is a typo, can you fix it?

Reply:
True!! Well seen!
I also correct it in the KeyPhrases prompt

Reply:
Fixed!